src/OptionsStruct.jl

```diff
@@ -40,6 +42,7 @@ will be normalized to sum to 1.0 after initialization.
 function MutationWeights(;
     mutate_constant=0.048,
     mutate_operator=0.47,
+    swap_operands=0.024,
```

Comment:
```suggestion
    swap_operands=0.0,
```
Just to follow SemVer, we can introduce this initially at 0.0 for a patch update (0.23.1), so that existing deterministic searches do not change, and then later we can adjust the weight to this. Does that sound good?